.github/workflows/check_deps.yml

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
Same comment as the other PRs. Please ensure the standard templates are updated.